Dockerfile

```diff
@@ -1,13 +1,13 @@
 FROM docker.io/library/golang:1.18.1-alpine as builder
+ARG TARGETPLATFORM
 RUN apk add git make
 ENV ORASPKG /oras
 ADD . ${ORASPKG}
 WORKDIR ${ORASPKG}
-RUN make build-linux
-RUN mv ${ORASPKG}/bin/linux/amd64/oras /go/bin/oras
+RUN make "build-$(echo $TARGETPLATFORM | sed 's/\//-/g')"
+RUN mv ${ORASPKG}/bin/${TARGETPLATFORM}/oras /go/bin/oras
 
 FROM docker.io/library/alpine:3.13.5
```

Comment:
Do we want to upgrade the alpine version like `3.15.4`?

Reply:
Upgraded to the latest.